docs/HOWTO -- NVIDIA AGX ORIN  on-device kernel build.md

```diff
@@ -0,0 +1,112 @@
+# HOWTO -- NVIDIA AGX ORIN on kernel on-device development process
+
+
+Nvidia is powerful device so developing and something lowlevel as drivers and passthroughs may be fast and efficient when done on board directly.
+
+
+1. Preparational steps -- get a device, plug in monitor and keyboard (in most cases you need DP cable and monitor with DP socket, adapters DP-HDMI will not work before firmware update).
+2. Boot and update onboard L4T ubuntu based linux distribution -- here are the link to the nvidia instructions -- .
```

Comment:
**Ubuntu-based Linux** distribution 

And what exactly link should we use here?

P.S. If I use another distribution, what should I do? Can we make this sentence more general for all environments? I am not sure if it necessary, but I should ask.

Reply:
This is actually quite confusing as our common baseline is not Ubuntu-based but NixOS-based ghaf.
Instead following this route of on-device development any further - this on-device kernel development process should be updated to NixOS. We have some good inputs on that already so let's do that instead.